backend/tests/v5/e2e/routes/teamspaces/projects/models/common/tickets.test.js

```diff
@@ -204,6 +205,7 @@ const testAddTicket = () => {
 	describe('Add ticket', () => {
 		const { users, teamspace, project, con, fed } = generateBasicData();
 		const template = ServiceHelper.generateTemplate();
+		const statusValues = times(5, () => ({ name: ServiceHelper.generateRandomString(), type: statusTypes[0] }));
```

Comment:
maybe make the type variable too so we're sure all the cases are covered?

```suggestion
		const statusValues = times(statusTypes.length, (i) => ({ name: ServiceHelper.generateRandomString(), type: statusTypes[i] }));
```